### DISCUSSION TITLE
How to exclude projects from apphost publish

### DISCUSSION THREAD

**Discussion by Lindsay-Mathieson** [microsoft/aspire#4400] [Q&A] [answered]
I have an aspire solution with a cloud api and some web apps, but also some local windows service client projects that should not be published to the cloud (Azure in this case).

How do I exclude them from the apphost publish?

**Comment by davidfowl** [comment, marked as answer]
[ExcludeFromManifest](https://learn.microsoft.com/en-us/dotnet/api/aspire.hosting.resourcebuilderextensions.excludefrommanifest?view=dotnet-aspire-8.0)

**Reply by Lindsay-Mathieson** [reply]
Thanks!
